<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCaseInformation</class>
 <widget class="QDialog" name="DialogCaseInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1486</width>
    <height>1025</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: dark-gray;
color: white;
}
QLabel{
font: 60pt &quot;Helvetica Neue&quot;;
}
QWidget{
background-color: dark-gray;
color: white;
}
QLabel{
font: 60pt &quot;Helvetica Neue&quot;;
}
QFrame{
border: 0px;
}
QPushButton{
font: bold 60px;
qproperty-flat: true;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frameTitle">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="labelTitle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777214</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>CASE INFORMATION</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameInfo">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelPhysicianName">
        <property name="minimumSize">
         <size>
          <width>651</width>
          <height>91</height>
         </size>
        </property>
        <property name="text">
         <string>Physician Name*</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditPhysitianName">
        <property name="minimumSize">
         <size>
          <width>611</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPatientId">
        <property name="minimumSize">
         <size>
          <width>651</width>
          <height>91</height>
         </size>
        </property>
        <property name="text">
         <string>Patient ID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditPatientId">
        <property name="minimumSize">
         <size>
          <width>611</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelLocation">
        <property name="minimumSize">
         <size>
          <width>651</width>
          <height>91</height>
         </size>
        </property>
        <property name="text">
         <string>LOCATION</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditLocation">
        <property name="minimumSize">
         <size>
          <width>611</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelDateAndTime">
        <property name="minimumSize">
         <size>
          <width>651</width>
          <height>91</height>
         </size>
        </property>
        <property name="text">
         <string>DATE &amp; TIME</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditDateAndTime">
        <property name="minimumSize">
         <size>
          <width>611</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelInto">
        <property name="minimumSize">
         <size>
          <width>721</width>
          <height>101</height>
         </size>
        </property>
        <property name="text">
         <string>*REQUIRED FIELD</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>545</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonNext">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>NEXT</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonNext</sender>
   <signal>clicked()</signal>
   <receiver>DialogCaseInformation</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1383</x>
     <y>875</y>
    </hint>
    <hint type="destinationlabel">
     <x>742</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
